eng/performance/benchmark_jobs.yml

```diff
@@ -90,16 +90,18 @@ jobs:
         container: ${{ parameters.container }}
         strategy:
           matrix:
-            ${{ each framework in parameters.frameworks }}:
-              ${{ framework }}:
-                _Framework: ${{ framework }}
+            ${{ each channel in parameters.channels }}:
+              ${{ channel }}:
+                _Channel: ${{ channel }}
                 _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
                 _PerfLabArguments: $(PerfLabArguments)
-                _BuildConfig: ${{ parameters.architecture }}_$(_Framework)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
+                _BuildConfig: ${{ parameters.architecture }}_$(_Channel)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
         steps:
         - checkout: self
           clean: true
-        - script: $(Python) scripts/ci_setup.py --frameworks $(_Framework) --architecture ${{parameters.architecture}} --perf-hash $(Build.SourceVersion) --queue ${{parameters.queue}} --build-number $(Build.BuildNumber) --build-configs $(_Configs)
+        - script: $(Python) scripts/channel_map.py --channel $(_Channel) --setup-pipeline # find the channel's matching tfm 
+          displayName: Look up TFM
+        - script: $(Python) scripts/ci_setup.py --channels $(_Channel) --architecture ${{parameters.architecture}} --perf-hash $(Build.SourceVersion) --queue ${{parameters.queue}} --build-number $(Build.BuildNumber) --build-configs $(_Configs)
```

Comment:
This isn't a problem with your PR but since you're changing this line can you add a `displayName` here also? I dislike that it shows up as "CmdLine" in AzDo

Reply:
yah np